manifests/init.pp

```diff
@@ -63,13 +56,13 @@
   }
 
   # Validate OS
-  case $::operatingsystem {
+  case $::lsbdistid {
```

Comment:
This set of changes, moving from operatingsystem* to lsdist* vars, is what's currently breaking linting tests. The VPP tests at the very bottom of spec/class/opendaylight_spec.rb assume the old vars. Is there some reasoning for using these new vars, vs the old ones? Not a big deal, both seem to work, but I don't know the background.